README.md

```diff
@@ -84,3 +84,8 @@ altered for customization. If some tweaking is required, simply create
 a new configuration file with a ```.local.conf``` suffix and some non
 zero prefix like ```20-user.local.conf```. Files using this suffix are
 on the ```.gitignore``` and not handled as untracked or dirty.
+
+## Contribution
+
+Every help is appreciated, for some guidelines and recommendations check
```

Comment:
both sounds good, will adjust and also move it to a CONTRIBUTION.md file inside our repo